lib/src/util/toastLevels.ts

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
Don't forget to delete this

Reply:
> Don't forget to delete this

Removed here: [3926694](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/3926694bfc214d1a3ac68fa74140b39e78463c21)